dspace-api/src/main/java/org/dspace/authorize/dao/ResourcePolicyDAO.java

```diff
@@ -79,4 +80,33 @@
      */
     public List<ResourcePolicy> findByDSoAndActionExceptRpType(Context c, DSpaceObject o, int actionID,
             String rpType) throws SQLException;
+
```

Comment:
@Micheleboychuk our contribution rules require javadoc for all the new public methods. Please add here and in all the other public methods in this interface